.github/workflows/test.yml

```diff
@@ -20,10 +20,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tag: ['2.12']
+        tag: ['2.18']
 
     container:
-      image:  google/dart:${{ matrix.tag }}
+      image:  dart:${{ matrix.tag }}
```

Comment:
Is this shorthand syntax for a dart age that github actions "knows" about or did google publish a top-level image for dart?

Reply:
Google retired google/dart, it hasn't had a new version of dart in a while.

So yes dart Is the new top level.